app.json

```
@@ -11,15 +11,14 @@
       "required": true
     }
   },
-  "formation": {
-  },
-  "addons": [
-
-  ],
+  "formation": {},
+  "addons": [],
   "buildpacks": [
     {
       "url": "heroku/nodejs"
+    }, {
+      "url": "heroku-community/apt"
     }
```

```suggestion
  "buildpacks": [
    {
      "url": "heroku-community/apt"
    },
    {
      "url": "heroku/nodejs"
    }
```

I'm not sure it would matter in this case, but from the [Heroku docs for multiple buildpacks](https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app):

> The buildpack for the **primary language** of your app should always be the **last** buildpack in the list. This ensures that defaults for that primary language are applied instead of those for another language, and allows Heroku to correctly detect the primary language of your app.